app/views/courses/_description_content.html.erb

```diff
@@ -0,0 +1,127 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h3 class="govuk-heading-m govuk-!-font-size-27">
+      <%= link_to 'About this course', manage_ui_course_page_url(provider_code: @provider[:provider_code], course_code: @course.attributes[:course_code]) + '/about', class: 'govuk-link' %>
+    </h3>
+    <dl class="govuk-summary-list govuk-summary-list--short">
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          About this course
+        </dt>
+        <dd class="govuk-summary-list__value govuk-summary-list__value--truncate <%= 'course-parts__fields__value--empty' if @course.attributes[:about_course].nil? %>" data-qa="course__about">
+          <%= @course.attributes[:about_course] || 'Empty' %>
+        </dd>
+      </div>
+
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          Interview process (optional)
+        </dt>
+        <dd class="govuk-summary-list__value govuk-summary-list__value--truncate <%= 'course-parts__fields__value--empty' if @course.attributes[:interview_process].nil? %>" data-qa="course__interview_process">
+          <%= @course.attributes[:interview_process] || 'Empty' %>
+        </dd>
+      </div>
+
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          How school placements work
+        </dt>
+        <dd class="govuk-summary-list__value govuk-summary-list__value--truncate <%= 'course-parts__fields__value--empty' if @course.attributes[:how_school_placements_work].nil? %>" data-qa="course__placements_info">
+          <%= @course.attributes[:how_school_placements_work] || 'Empty' %>
+        </dd>
+      </div>
+    </dl>
+
+    <h3 class="govuk-heading-m">
+      <% if @course.attributes[:funding] == 'fee' %>
+        <%= link_to 'Course length and fees', manage_ui_course_page_url(provider_code: @provider[:provider_code], course_code: @course.attributes[:course_code]) + '/fees-and-length', class: 'govuk-link' %>
+      <% else %>
+        <%= link_to 'Course length and salary', manage_ui_course_page_url(provider_code: @provider[:provider_code], course_code: @course.attributes[:course_code]) + '/salary', class: 'govuk-link' %>
+      <% end %>
+    </h3>
+    <dl class="govuk-summary-list govuk-summary-list--short">
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key ">
+          Course length
+        </dt>
+        <dd class="govuk-summary-list__value govuk-summary-list__value--truncate <%= 'course-parts__fields__value--empty' if @course.attributes[:course_length].nil? %>" data-qa="course__length">
+          <%= course_length(@course) || 'Empty' %>
+        </dd>
+      </div>
+      <% if @course.attributes[:funding] == 'fee' %>
```

Comment:
It might be nice if we could "ask" the course if it has a fee. Maybe `@course.has_fees?` or something like that?